docker-compose.yml

```diff
@@ -20,9 +20,6 @@ services:
       - "8000:8000"
     depends_on:
       - postgres
-  documentation:
-    restart: always
-    build: ./
     command: >
```

Comment:
You need to delete [lines 23-28](https://github.com/hsv-dot-beer/hsvdotbeer/blob/cbefb61df7763d547d2f95150fee7a15cfe95c58/docker-compose.yml#L23-L28) too